CHANGELOG.md

```diff
@@ -34,6 +40,15 @@ backend {
 }
 ```
 
+### Runtime Attributes
+
+#### GPU Attributes
+
+* The `gpuType` attribute is no longer validated against a whitelist at workflow submission time. Instead, validation now happens at runtime. This allows any valid accelerator to be used.
+* The `nvidiaDriverVersion` attribute is now available in WDL `runtime` sections. The default continues to be `390.46` which applies if and only if GPUs are being used.
+* A default `gpuType` ("nvidia-tesla-p100") will now be applied if `gpuCount` is specified but `gpuType` is not.
```

Comment:
I noticed this model is nearer the top of the price range [1] - if a user just wants "a GPU" would it make sense to give them the most basic model?

[1] https://cloud.google.com/compute/pricing#gpus

Reply:
Oops, in my defense the link I clicked took me to a view that did not have "don't look at me" visible

Reply:
Still a valid point though. It seems that k80 would be the better default value here